packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -654,16 +654,26 @@ const ProductList = (props) => {
                                                               }}
                                                               dynamicImageProps={{
                                                                   widths: [
-                                                                      '50vw',
-                                                                      '50vw',
-                                                                      '20vw',
-                                                                      '20vw',
-                                                                      '25vw'
+                                                                      // Each product image can take up the full 50% of the screen width
+                                                                      '50vw', // base <= 479px
+                                                                      '50vw', // sm >= 480px ; <= 767px
```

Comment:
It does feel good to move away from the `density` logic that was here in the previous iteration 👍

Reply:
I'm not quite sure yet. The unconditional addition of images for double density and thus having no influence on the behavior at all is not necessarily the real thing either.